Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM ubuntu:20.04 AS container-tools
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && apt-get install -y wget git make \
-	libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config
+	libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config \
+        file
 
-RUN wget -P /tmp https://go.dev/dl/go1.18.linux-amd64.tar.gz && \
-	tar -C /usr/local -xzf /tmp/go1.18.linux-amd64.tar.gz
+# Detect host architecture
+RUN file /bin/bash | grep -q x86-64 && echo amd64 > /tmp/arch || true
+RUN file /bin/bash | grep -q aarch64 && echo arm64 > /tmp/arch || true
+
+RUN wget -P /tmp https://go.dev/dl/go1.18.linux-$(cat /tmp/arch).tar.gz && \
+	tar -C /usr/local -xzf /tmp/go1.18.linux-$(cat /tmp/arch).tar.gz
```

Comment:
mind blown

Reply:
@EmbeddedAndroid it seems this got removed (the wget line) in your latest update?